OpenEphys.Onix/OpenEphys.Onix/NeuropixesV1eDeviceInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace OpenEphys.Onix
+{
+    class NeuropixesV1eDeviceInfo : DeviceInfo
```

Comment:
```suggestion
    class NeuropixelsV1eDeviceInfo : DeviceInfo
```

Probably easiest to rename the whole file to fix this typo.